src/persistence/job.js

```diff
@@ -63,5 +63,11 @@ module.exports = {
         SELECT * FROM jobs
         `);
     return rows;
+  },
+  async paginationGet(offset, limit) {
```

Comment:
Please set the default values because we don't make sure that the `offset` and `limit` is always provided when calling `paginationGet` at other places